as_perimenoun_oi_gynaikes.ass

```diff
@@ -541,13 +542,13 @@ Dialogue: 0,0:40:33.84,0:40:36.11,Default,,0,0,0,,Ask for an explanation?\N But
 Dialogue: 0,0:40:36.11,0:40:37.44,Default,,0,0,0,,PASOK and New Democracy?!
 Dialogue: 0,0:40:37.44,0:40:38.54,Default,,0,0,0,,Two faces of the same coin!
 Dialogue: 0,0:40:38.54,0:40:39.89,Default,,0,0,0,,I feel sorry for you!
-Dialogue: 0,0:40:39.89,0:40:42.94,Default,,0,0,0,,In Cassandra you piss, in Volvi you take the piss!
+Dialogue: 0,0:40:39.89,0:40:42.94,Default,,0,0,0,,In Cassandra you piss, in Volvi you unload!

```

Comment:
"Take the piss" is funnier and more fitting here.

Reply:
It is indeed weird how to translate that. My assumption is that it refers to cumming and not pissing but ok :)

Reply:
This was one of the hardest lines to translate. The translation does not come from me but from a youtube user (TEL9021) who pointed it out. I really liked it because the phrase "Taking the piss" is urologically funny just as the original Greek phrase. They of course do not match 100% (you are right, the original Greek refers to pissing in one place and cumming in another), but I think we can't do better than that.